tests/core/data/test_data_repositories.py

```diff
@@ -52,9 +52,9 @@ def test_load_all_with_filters(self, data_node, repo, init_sql_repo):
 
         for i in range(10):
             data_node.id = DataNodeId(f"data_node-{i}")
-            data_node.name = f"data_node-{i}"
```

Comment:
Why do we stop using `name` here? It should still work, shouldn't it?

Reply:
not exactly, because it's not an attribute of dn anymore, and so when we try to find it with filtering in `_load_all` and `_search` for sql_repo, it returns an error as it's not a column in dn table.

Reply:
Oh... ok got it.